File: Manual 01/Question01.cpp
#include <iostream>
using namespace std;

int main()
{
    int a, b, c, d, e;

    // Input values for a, b, c, d, e
    cout << "Input the value of a: ";
    cin >> a;
    cout << "Input the value of b: ";
    cin >> b;
    cout << "Input the value of c: ";
    cin >> c;
    cout << "Input the value of d: ";
    cin >> d;
    cout << "Input the value of e: ";
    cin >> e;

    // Determine the largest integer
    if (a > b && a > c && a > d && a > e)
    {
        cout << "a is the largest integer" << endl;
    }
    else if (b > a && b > c && b > d && b > e)
    {
        cout << "b is the largest integer" << endl;
    }
    else if (c > a && c > b && c > d && c > e)
    {
        cout << "c is the largest integer" << endl;
    }
    else if (d > a && d > b && d > c && d > e)
    {
        cout << "d is the largest integer" << endl;
    }
    else if (e > a && e > b && e > c && e > d)
    {
        cout << "e is the largest integer" << endl;
    }

    // Determine the smallest integer
    if (a < b && a < c && a < d && a < e)
    {
        cout << "a is the smallest integer" << endl;
    }
    else if (b < a && b < c && b < d && b < e)
    {
        cout << "b is the smallest integer" << endl;
    }
    else if (c < a && c < b && c < d && c < e)
    {
        cout << "c is the smallest integer" << endl;
    }
    else if (d < a && d < b && d < c && d < e)
    {
        cout << "d is the smallest integer" << endl;
    }
    else if (e < a && e < b && e < c && e < d)
    {
        cout << "e is the smallest integer" << endl;
    }

    return 0;
}

File: JavaScript/Manual 01/Question01.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

let numbers = [];

rl.question("Enter the first integer: ", function(num1) {
    numbers.push(parseInt(num1));

    rl.question("Enter the second integer: ", function(num2) {
        numbers.push(parseInt(num2));

        rl.question("Enter the third integer: ", function(num3) {
            numbers.push(parseInt(num3));

            rl.question("Enter the fourth integer: ", function(num4) {
                numbers.push(parseInt(num4));

                rl.question("Enter the fifth integer: ", function(num5) {
                    numbers.push(parseInt(num5));

                    // Initialize largest and smallest
                    let largest = numbers[0];
                    let smallest = numbers[0];

                    // Find the largest and smallest numbers
                    for (let i = 1; i < numbers.length; i++) {
                        if (numbers[i] > largest) {
                            largest = numbers[i];
                        }
                        if (numbers[i] < smallest) {
                            smallest = numbers[i];
                        }
                    }

                    // Output the results
                    console.log("The largest number is: " + largest);
                    console.log("The smallest number is: " + smallest);

                    rl.close();
                });
            });
        });
    });
});

